### PR TITLE
optional arg for ordering subcourseSearch results descending

### PR DESCRIPTION
**Pull request by Fabian5150**
Added optional argument to the subcourseSearch query to provide a option for 'orderBy'.
This was necessary for the Screening Tool, as it used to fetch only the oldest few courses, which made it a bit useless

### CODE REVIEW

File: graphql/subcourse/fields.ts [hunk 1]
```diff
@@ -148,14 +148,16 @@ export class ExtendedFieldsSubcourseResolver {
         @Arg('search') search: string,
         @Arg('courseStates', () => [String], { nullable: true }) courseStates: course_coursestate_enum[],
         @Arg('take', () => GraphQLInt) take: number,
-        @Arg('skip', () => GraphQLInt, { nullable: true }) skip: number = 0
+        @Arg('skip', () => GraphQLInt, { nullable: true }) skip: number = 0,
+        @Arg('orderDesc', { nullable: true }) orderDesc: boolean
```
Comment (Jonasdoubleyou) [inline graphql/subcourse/fields.ts:152]: We could potentially just expose the Prisma OrderBy type here (there should be a generated GraphQL Type for it) - Then we are flexible towards future requirements

**Review by Jonasdoubleyou — APPROVED** [review]

File: graphql/subcourse/fields.ts (hunk 1 above)
Comment (Jonasdoubleyou) [reply graphql/subcourse/fields.ts:152]: Also maybe rather orderByLastUpdate or something like that? orderDesc is a bit generic

Comment (Jonasdoubleyou) [reply graphql/subcourse/fields.ts:152]: Or maybe orderBy: "last-update", then we can add further sorting variants in the future?

Comment (Fabian5150) [reply graphql/subcourse/fields.ts]: Went for the third option now, sounds like a good idea.

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**